templates/mail/notify.txt

```diff
@@ -1,22 +1,26 @@
 Hi cloud.gov user,
 
-cloud.gov frequently updates the programming language buildpacks available to our customers, and we recently updated 
-{{- if .IsMultipleApp}} the buildpacks that your applications use. {{else}} the buildpack that your application uses. {{end -}}
-Buildpack updates include programming language updates and often include security fixes.
-
-To take advantage of buildpack updates, you should restage your application{{if .IsMultipleApp}}s{{end}}. Documentation for this process: https://cloud.gov/docs/getting-started/app-maintenance/
-
-You can restage apps by opening the command line and entering the following command: cf restage YOURAPPNAME
+cloud.gov frequently updates the programming language buildpacks available to
+our customers. Buildpack updates include programming language updates and 
+often include security fixes.
 {{if .IsMultipleApp}}
-The following applications have updated buildpacks available:
+We recently updated buildpacks in use by your applications. To take advantage
+of the update, you should restage your applications. You can do this by
+opening the command line and entering the following commands:
 {{else}}
-The following application has an updated buildpack available:
+We recently updated the buildpack in use by your application. To take 
```

Comment:
Is it *should* or *must*?

Reply:
Should we indicate that this is best done first in a development or staging environment?

Reply:
It's a _should_. I'd support adding further details about dev/staging to the docs page, along with this note about downtime: https://github.com/18F/cg-site/pull/1118 - the email should be pretty short/minimal, with detailed guidance in the docs.

Reply:
If it's a _should_ then I strongly suggest we include that in the email, and not rely on folks to go to the docs page to figure that out.